### ISSUE TITLE
[@loaders.gl/video] window is not undefined error 

### ISSUE THREAD

**Issue by chrisgervang** [visgl/loaders.gl#2164]
Removing this safe guard for `window` in the `.ts` rewrite introduced a bug in server side rendering when upgrading past version [v3.1.0-alpha.5](https://github.com/visgl/loaders.gl/releases/tag/v3.1.0-alpha.5), since the module is accessing window and it is not defined.

Last working stable version appears to be [3.0.12](https://github.com/visgl/loaders.gl/releases/tag/v3.0.12) 

@hkim8567 reported an error today from his next.js logs after a minor package upgrade: 
```
error - ReferenceError: window is not defined
ReferenceError: window is not defined
    at Object.<anonymous> (***/node_modules/@loaders.gl/video/dist/es5/lib/gifshot/gifshot.js:8:8)
```
This traces back to this code: `URL: window.URL || window.webkitURL || window.mozURL || window.msURL,`

_Originally posted by @chrisgervang in https://github.com/visgl/loaders.gl/pull/1930#discussion_r860111045_

**Comment by chrisgervang** [comment]
Current workaround in package.json:
```
"resolutions": {
  "@loaders.gl/video": "3.0.12"
}
```

**Comment by belom88** [comment]
Included in v3.2.5
